cloudinit/sources/helpers/vultr.py

```diff
@@ -202,7 +220,30 @@ def generate_private_network_interface(interface):
         ]
     }
 
+    # Options that may or may not be used
+    if "mtu" in interface:
+        netcfg['mtu'] = interface['mtu']
+
+    if "accept-ra" in interface:
+        netcfg['accept-ra'] = interface['accept-ra']
+
+    if "routes" in interface:
+        netcfg['subnets'][0]['routes'] = interface['routes']
+
     return netcfg
 
 
+# Make required adjustments to the network configs provided
+def do_network_interfaces(interfaces):
```

Comment:
Can we make this name more descriptive? `add_interface_names` or something similar?

Reply:
Sure can. Was having a hard time naming that one as I wasn't sure if there was going to be anything else. At this time, likely not! Changed.